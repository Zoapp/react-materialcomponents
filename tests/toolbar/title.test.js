import React from "react";
import renderer from "react-test-renderer";
import Title from "libs/toolbar/title";

describe("toolbar/Title", () => {
  it("can render children", () => {
    const component = renderer.create(<Title>Children are here!</Title>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
