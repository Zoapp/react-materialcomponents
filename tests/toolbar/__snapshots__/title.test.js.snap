// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toolbar/Title can render children 1`] = `
<span
  className="mdc-toolbar__title"
>
  Children are here!
</span>
`;
